doc/governance.rst

```diff
@@ -0,0 +1,126 @@
+with their project related activities by giving them direct access to the
+project’s repository and is represented as being an organization member on the
+scikit-learn `GitHub organization <https://github.com/orgs/scikit-learn/people>`_.
+Core developers are expected to review code
+contributions, can merge approved pull requests, can cast votes for and against
+merging a pull-request, and can be involved in deciding major changes to the
+API.
+
+New core developers can be nominated by any existing core developers. Once they
+have been nominated, there will be a vote by the current core developers.
+Voting on new core developers is one of the few activities that takes place on
+the project's private management list. While it is expected that most votes
+will be unanimous, a two-thirds majority of the cast votes is enough. The vote
+needs to be open for at least 1 week.
+
+Core developers that have not contributed to the project (commits or GitHub
+comments) in the past 12 months will be asked if they want to become emeritus
+core developers and recant their commit and voting rights until they become
+active again. The list of core developers, active and emeritus (with dates at
+which they became active) is public on the scikit-learn website.
+
+Technical Committee
+-------------------
+The Technical Committee (TC) members are core developers who have additional
+responsibilities to ensure the smooth running of the project. TC members are expected to
+participate in strategic planning, and approve changes to the governance model.
+
+Membership of the TC is by nomination by a core developer. A nomination will
+result in discussion which cannot take more than a month and then a vote by
+the core developers which will stay open for a week. TC membership votes are
+subject to a two-third majority of all cast votes as well as a simple majority
+approval of all the current TC members. TC members who do not actively engage
+with the TC duties are expected to resign.
+ 
+The initial Technical Committee of scikit-learn consists of :user:`Alexandre Gramfort <agramfort>`,
+:user:`Olivier Grisel <ogrisel>`, :user:`Andreas Müller <amueller>`, :user:`Joel Nothman <jnothman>`,
+:user:`Hanmin Qin <qinhanmin2014>`, :user:`Gaël Varoquaux <GaelVaroquaux>`, and
+:user:`Roman Yurchak <rth>`.
+ 
+Decision Making Process
+=======================
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project contributors’ `mailing list <mailto:scikit-learn@python.org>`_
+and the `issue tracker <https://github.com/scikit-learn/scikit-learn/issues>`_.
+Occasionally, sensitive discussion occurs on a private list.
+
+Scikit-learn uses a "consensus seeking" process for making decisions. The group
+tries to find a resolution that has no open objections among core developers
+within a month. If no consensus can be reached in this time frame, a
```

Comment:
When I tried to fix this issue on the doc version, I failed to keep it simple, but it was along the lines of: "A deadline is set once two core developers call for a deadline on a particular issue."

Reply:
This is indeed a tricky point and I'm not sure how to best address it.
Go has (shepherds)[https://github.com/golang/proposal] that can call for votes if I understand correctly. I'm not sure how well that works.
Having a call for vote and a seconding would also work. Can the proposer call for a vote and/or second?

Reply:
I'm not sure what you're asking in

> Can the proposer call for a vote and/or second?

I'm fine with two people proposing a vote, but it seems absent here.

Reply:
```
... The group
tries to find a resolution that has no open objections among core developers
within a month. ...
```
->
```
... The group
tries to find a resolution that has no open objections among core developers.
At any given time, if a core developer suggests a deadline for the discussion
and another core developer seconds the suggestion, that deadline is the time
the team has to reach a consensus. ...
```

Reply:
@jnothman The question was whether to do a vote, we need two people that are not the proposer to call for a vote, or if the proposer can be one of the two people.

I'll try to reformulate this in a bit.

Reply:
> within a month of what? How/when does one declare that a decision needs to be made?  Does one create a SLEP to force a decision, such that then there is a month from its posting to find consensus?

According to this document the SLEPs are explicitly used to call for a vote. So that seems like a natural solution to me. But it doesn't have to be a strict requirement, just a possibility.

How about:
- call for a vote = vote will take place in 1 month.
- any call for a vote must be backed by a SLEP
- any core dev can call for a vote, regardless of who created the SLEP
- SLEP creation != call for a vote, though the proposer can do both at the same time